test/karma-conf.js

```diff
@@ -240,7 +240,7 @@ module.exports = function (config) {
         browsers = Object.keys(browserStackBrowsers);
     }
 
-    const browserCount = argv.browsercount || (os.cpus().length - 1);
+    const browserCount = argv.browsercount || (os.cpus().length - 2);
```

Comment:
What if the os has only CPUs available? Shouldn't it be something like this?
`const browserCount = argv.browsercount || Math.max(1, os.cpus().length - 2);`

Reply:
You are right. I corrected it to the above.